### PR TITLE
Raw Brainstorming Version of 24 Pull Requests Ruby

### PR DESCRIPTION
**Pull request by kotp**
Still has some work to do.  Please let me know what we need.  This 24PullRequests has started a few hours ago...

@kytrinyx Anything you can think of?

### CODE REVIEW

**Comment by kytrinyx** [comment]
Does this link help at all? https://github.com/exercism/exercism.io/blob/master/docs/getting-involved-in-a-track.md

**Comment by kytrinyx** [comment]
Low hanging fruit: http://exercism.io/languages/ruby/todo

**Comment by Insti** [comment]
Maybe something about solve some problems on Exercism, and report back with things that could make the experience better.

A lot of tests need better failure messages.

File: docs/24pullrequests.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Look for the following labels in our issue list:
+
+* first-timers only
+* good first patch
+* bug
+* enhancement
+
+Or notice something to contribute?  Documentation not quite up to par?
+Something missing?  Submitting and reviewing exercises on Exercism and
+notice something you don't like?  Create a pull request!
```
Comment (kotp) [inline docs/24pullrequests.md]: Like this, @Insti?

Comment (Insti) [inline docs/24pullrequests.md]: looks good.

**Pull request merged**